src/components/ReportActionItem/ReportActionItemImage.js

```diff
@@ -60,7 +61,7 @@ function ReportActionItemImage({thumbnail, image, enablePreviewModal, transactio
                 <EReceiptThumbnail transactionID={transaction.transactionID} />
             </View>
         );
-    } else if (thumbnail && !isLocalFile) {
+    } else if (thumbnail && !isLocalFile && !Str.isPDF(imageSource)) {
```

Comment:
@namhihi237 Why is this change necessary?

Reply:
because with PDF we use the local thumbnail preview

Reply:
This fix loading Indicator is displayed happen on desktop.

Reply:
[This case](https://github.com/Expensify/App/issues/33412#issuecomment-1866338538) might be related. But after clearing the Onyx Database, it was not reproducible.

Reply:
I can reproduce it on the desktop. so we need to fix  here.